frontend/src/libs/components/radio/radio.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  type Control,
+  type FieldErrors,
+  type FieldPath,
+  type FieldValues,
+} from 'react-hook-form';
+
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useFormController } from '~/libs/hooks/hooks.js';
+
+import common from './common.module.scss';
+import radio from './radio.module.scss';
+
+const styles = { ...common, ...radio };
+
+type Properties<T extends FieldValues, N extends FieldPath<T>> = {
+  control: Control<T, null>;
+  errors?: FieldErrors<T>;
```

Comment:
Can we make it required?
```suggestion
  errors: FieldErrors<T>;
```

Reply:
@PavloSukhinin yes, but for radio button it makes no sense, because only the last one is required. You can't have a single radio element which is invalid because they all share the same name thus have single source of error.
In other words, either they all have errors, or none of them. But only the last one will display error after itself.